infoblox-cni-install.yaml

```diff
@@ -42,6 +43,12 @@ spec:
       containers:
       - name: install-cni
         image: infoblox/infoblox-cni-install
+		   env:
```

Comment:
You should correct the space here. I think tab space is used instead of normal space character for that line alone.

Reply:
done